src/frontend/src/handler/drop_source.rs

```diff
@@ -114,4 +135,23 @@ mod tests {
                 .to_string()
         );
     }
+
+    #[tokio::test]
+    async fn test_drop_materialized_view_using_drop_source() {
```

Comment:
We may write these tests as planner tests under `src/frontend/test_runner/tests/testdata` so they can be more intuitive and easy to update.